app/main.py

```diff
@@ -1,6 +1,26 @@
+import requests
+import os
+
+from dotenv import load_dotenv
+
+load_dotenv()
+
+
 def get_weather() -> None:
-    # write your code here
-    pass
+    api_key = os.getenv("API_KEY")
```

Comment:
Consider using a constant for the API URL and the filtering value. According to the checklist, constants should be used for such values and should be in uppercase. For example, `API_URL` and `FILTERING` could be defined at the top of the file.